### DISCUSSION TITLE
Customize UI scaling

### DISCUSSION THREAD

**Discussion by eFe-Munoz** [friction2d/friction#109] [Ideas]
I think it will bei great and necessary to have an option by percentage of UI scaling, on high resolution monitors texts and icons are visualizedb tiny.

Thanks and congratulations for the application.

**Comment by rodlie** [comment]
Is it a problem? as default Friction will adapt to whatever your desktop is set to, and you can override if wanted in preferences.

27" 4k with 150% scaling at work:
![Screenshot from 2023-12-19 10-36-24](https://github.com/friction2d/friction/assets/34516798/14bd8c2a-636c-405c-9c2a-4df05c946fd9)

Looks ok to me(?).

Preferences:
![Screenshot from 2023-12-19 10-38-13](https://github.com/friction2d/friction/assets/34516798/b83d7a7e-a0fe-442a-b184-e385dd951a4c)

**Comment by eFe-Munoz** [comment]
Sorry, maybe I have an old version? I have installed 0.9.4. And I see this in Preferences > Canvas:

![imagen](https://github.com/friction2d/friction/assets/92168582/a6b6cb2f-f051-4203-8399-d722ae1c9b1b)

**Comment by rodlie** [comment]
Ah, yes it was introduced in 0.9.5rc1 :) Will do a proper release this week.

**Comment by eFe-Munoz** [comment]
Great news! thanks!
